src/openapi/streaming/subscription.js

```diff
@@ -120,6 +139,17 @@ function performAction(action) {
             }
             break;
 
+        case ACTION_MODIFY_PATCH:
+            switch (this.currentState) {

```

Comment:
Then we would be in transition state, performAction is only called when we are not in a transition state